themes/default/data/newsroom.toml

```diff
@@ -1,4 +1,9 @@
 [[releases]]
+date = "18th October 2021"
+title = "Pulumi Ushers in the Developer-first Infrastructure Era with New Package Registry Powering Software Sharing and Reuse"
+summary = "Pulumi Ushers in the Developer-first Infrastructure Era with New Package Registry Powering Software Sharing and Reuse"
+url = "http://info.pulumi.com/press-release/pulumi-registry"
+
 date = "30th September 2021"
```

Comment:
```suggestion
[[releases]]
date = "30th September 2021"
```

Reply:
☝️ I _think_ you need this, as each entry needs to have this same key (so it ends up in the `releases` list).

Reply:
lol sorry and thank you